daemon/daemon.go

```diff
@@ -730,7 +736,7 @@ func (daemon *Daemon) Shutdown() error {
 	}
 
 	if daemon.containers != nil {
-		logrus.Debug("starting clean shutdown of all containers...")
+		logrus.Debugf("starting clean shutdown of all containers in %d seconds...", daemon.configStore.ShutdownTimeout)
```

Comment:
The wording feels to me like we are only going to start shutting down the containers after `ShutdownTimeout` seconds have expired.

Maybe use something like `start clean shutdown of all containers with a %d seconds timeout...`?

cc @thaJeztah for better wording :sweat_smile: